server/src/main/scala/com/stratio/crossdata/server/actors/ServerActor.scala

```diff
@@ -135,6 +144,26 @@ class ServerActor(cluster: Cluster, xdContext: XDContext, config: ServerActorCon
       st.jobsById.get(JobId(requester, queryId)).get ! CancelJob
   }
 
+  private def addToClasspath(file : File) : Unit = {
+    if (file.exists) {
+      val method : Method = classOf[URLClassLoader].getDeclaredMethod("addURL", classOf[URL])
+
+      method.setAccessible(true)
+      method.invoke(ClassLoader.getSystemClassLoader, file.toURI.toURL)
```

Comment:
You might like to restore reflection accessibility flag after the "dirty work" has been finished `method.setAccessible(false)`